locale/en/docs/guides/diagnostics-flamegraph.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Diagnostics - Flame Graphs
+layout: docs.hbs
+---
+
+# Flame Graphs
+
+## What's a flame graph useful for?
+
+Flame graphs are a way of visualizing CPU time spent in functions. They can help you pin down where you spend too much time doing synchronous operations.
+
+## How to create a flame graph
+
+You might have heard creating a flame graph for Node.js is difficult, but that's not true (anymore).
```

Comment:
I support this a lot as a technical documentation for how to make a flame graph using only basic primitives. However, I wouldn't call this easy. I someone asked me how to make flame graphs easily I would defer to [0x](https://github.com/davidmarkclements/0x).

Reply:
Hehe, easy in comparison. It's a second mention of 0x, so it's time I added a mention. thx

Reply:
someone who hasn't heard its difficult now has, maybe this will make people nervous before they even started? I third the `0x` reference